barretenberg/cpp/src/barretenberg/protogalaxy/protogalaxy_verifier.cpp

```diff
@@ -211,60 +138,56 @@ bool ProtoGalaxyVerifier_<VerifierInstances>::verify_folding_proof(const std::ve
     auto vanishing_polynomial_at_challenge = combiner_challenge * (combiner_challenge - FF(1));
     auto lagranges = std::vector<FF>{ FF(1) - combiner_challenge, combiner_challenge };
 
+    auto next_accumulator = std::make_shared<Instance>();
+    next_accumulator->instance_size = accumulator->instance_size;
+    next_accumulator->log_instance_size = accumulator->log_instance_size;
+    next_accumulator->is_accumulator = true;
     // Compute next folding parameters and verify against the ones received from the prover
```

Comment:
this comment and the one below are no longer true right? should just be Compute next folding parameters.